interface/cli/main.go

```diff
@@ -52,3 +59,19 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func coreAddress(cfg *config.Config) (string, error) {
```

Comment:
This func be moved to config package (and docker_host should be parsed and load with `Load` Function)

Reply:
No this is something related to the CLI only, config is a package shared between the Core and CLI. Ultimately CLI should not access the config at all and only rely on the `Info` API but to fetch this api it needs an endpoint for the core address and a detection for this endpoint.

Reply:
But for now, it reads all data from the config and using it (so config is shared between cli and server). So this one should be handled there as well (as all other envs/variables used by CLI)

Reply:
but config on the Core don't rely on the host environment, this config does. We are reading the env variables from the host that the core will never have. I could agree to have that on the core if the core itself can give its own address and in that case we don't even need any docker env variable but this is not the case.

Other solution that might be the way to go is to actually inject the config when we run the CLI like that the CLI script will inject the address based on the docker machine and the CLI still get the data from the config but the config package doesn't need to have any dependency with the env from the host. Tell me what you think about that ?

Reply:
I'm not quite following the part:

> We are reading the env variables from the host that the core will never have. (....)
 any dependency with the env from the host.

I don't see reason these env variables shouldn't be read in config package (here is just simply read - same as in config), but it you feel this one should stay in `main.go` - i'm ok with this.

Reply:
Otherwise you can have a look at this modification https://github.com/mesg-foundation/core/compare/feature/docker-machine-support...feature/docker-machine-support-2

It's not adding support in the CLI but in the dev tools and we can see later about the support for the CLI especially if we move the CLI in node... which will force us not to use the config package

Reply:
Yes I see, is this only for dev purpose? Because you mention about DO in PR comment above so I guess it won't work with such script. So if this is only for dev it's better to be in dev-cli then.

Reply:
For my specific need it's for dev (I run all my docker stack in a VM). I think anyway it's a nice feature to have in the CLI in general to have the support of docker machine and DO and other but this can be documented with the `MESG_CLIENT_ADDRESS` env variable anyway so i'm fine to have this just for dev and merge this other branch inside this one to revert